cmd/notation/plugin/install.go

```diff
@@ -185,12 +185,21 @@ func installPlugin(ctx context.Context, inputPath string, inputChecksum string,
 			return err
 		}
 		defer rc.Close()
+		// check for '..' in file name to avoid zip slip vulnerability
+		for _, f := range rc.File {
+			if strings.Contains(f.Name, "..") {
+				return fmt.Errorf("file name in zip cannot contain '..', but found %q", f.Name)
+			}
+		}
```

Comment:
What if a file name is `foo..bar.txt`?

Reply:
We do not accept such file either.

Reply:
This was recommended by the `github-advanced-security`: https://github.com/notaryproject/notation/security/code-scanning/44

Reply:
It is over-kill but I guess we can fix it later when we got a real file with `..` in it.

Reply:
I see. Resolving this comment for now.